Makefile

```diff
@@ -85,6 +85,20 @@ test-heavy: build $(SAMPLEDATA)
 test-pcapingest: bin/$(ZEEKPATH)
 	@ZEEK="$(CURDIR)/bin/$(ZEEKPATH)/zeekrunner" go test -v -run=PcapPost -tags=pcapingest ./ppl/zqd
 
+.PHONY: test-postgres
+test-postgres: build 
+	@echo "pg test: $(PG_TEST)"
+	@ZTEST_PATH="$(CURDIR)/dist:$(CURDIR)/bin" \
+		ZTEST_TAG=postgres \
+		ZTEST_ENV="MIGRATIONS_DIR=$(CURDIR)/ppl/zqd/migrations PG_TEST=$(PG_TEST)" \
```

Comment:
I'm not sure about stuffing the test environment settings through this var, but don't have an immediate alternative to suggest. I guess I'm unsure that we really need to have the ztests run with a completely clear environment; we could just alter their `PATH` to ensure they pick up the test binaries first.

Reply:
FWIW I wasn't a huge fan of doing this either. I think I'm for setting the testenv to the host env- I do worry about carrying over $PATH. I can imagine a situation where for some reasons a build on a binary stops working (or gets placed somewhere else) and some test keeps failing because its picking up a binary that exists in your path, while you are not sure why your code changes are having no effect.

Reply:
> That said, I'm uneasy about using MIGRATIONS_DIR to access files outside a ztest's temporary directory. That doesn't feel right.

Do you have a solution for this? Perhaps inputs could have a symlink type that would symlink a directory- I don't think you would want to copy the entirety of the migrations dir to a test dir, especially as they get bigger.

Reply:
I like the env idea, I'll add that.

Reply:
Took a little work but I've officially killed all the ZTEST_ENV env variable!